### ISSUE TITLE
 ARIA attribute is not allowed: aria-orientation="horizontal" when role="group"

### ISSUE THREAD

**Issue by SavannahBourgeois** [OfficeDev/office-ui-fabric-react#11646]
### Environment Information

- **Package version(s)**: all
- **Browser and OS versions**: all

### Describe the issue:
OverflowSet allows items to have role="group" within them. However, OverflowSet always sets the aria-orientation on the items. According to the documentation on aria-orientation [(link)](https://www.w3.org/TR/wai-aria/#aria-orientation), it should not be set alongside role="group".

### Please provide a reproduction of the issue in a codepen:
Not in codepen but in onenote online...
Step 1: Open Onenote Online using edge browser.
Step 2: Navigate to “Edit Notebook” button by using tab key.
Step 3: Navigate to “Edit Note in browser field” menu-item by using tab key and select it.
Step 4: Navigate to Home tab by using ctrl+F6 and select it.
Step 5: Navigate to Insert tab by using arrow key and activate it.
Step 6: Navigate to “Table” button by using tab key and expand it.”
Step 7: Select row and column and press enter key.
Step 8: Navigate to “Table” tab by using “ctrl+F6” and select it.
Step 9: Open Developer tool bar (F12)
Step 10: Verify whether Elements must only use allowed ARIA attributes.

This is the line that sets the aria-orientation without checking the role it is setting [OverflowSetBase](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.base.tsx)

```
 <Tag aria-orientation={vertical ? 'vertical' : 'horizontal'} {...uniqueComponentProps} className={this._classNames.root} role={role}>

        {items && this._onRenderItems(items)}

        {overflowItems && overflowItems.length > 0 && this._onRenderOverflowButtonWrapper(overflowItems)}

      </Tag>
```

#### Actual behavior:

If you pass an item with role="group", aria-orientation is incorrectly set.

#### Expected behavior:

If you pass an item with role="group", aria-orientation should not be set. There should be a check to ensure we are not passing role="group" before setting the property.

### Documentation describing expected behavior

[Aria-orientation](https://www.w3.org/TR/wai-aria/#aria-orientation)

**Comment by xugao** [comment]
@SavannahBourgeois - i think this is a bug. but to make sure i resolve your issue, could you share the html snippet of the rendered OverflowSet for your repro case? (sorry having trouble following the steps you provided) I mainly want to confirm the roles you set on the `OverflowSet` and all the items inside the set

**Comment by SavannahBourgeois** [comment]
Hope the screenshot below helps. Essentially if you insert a tab in OneNote and then click on the Table ribbon tab, the buttons in the ribbon are set within an OverflowSet. Let me know if that still isn't clear.
![image](https://user-images.githubusercontent.com/42548121/72079484-dc01ea00-32af-11ea-9322-80cef4a8c70e.png)

**Comment by micahgodbolt** [comment]
yeah....this is a tough one. when does the control add aria roles values, and when do we leave that up to the user. 

I added orientation back when a bug when filed on our examples that vertical menus needed aria-orientation. Since horizontal/vertical was handled by a overflowSet prop, i decided to do it within the control.

Problem is that this assumes a certain role, which then assumes roles for its children....which are both bad assumptions as those roles aren't always correct. 

So...my current take is that overflowSet is a utitlity component, and you need to add explicit aria tags for everything...then in commandBar etc, we pass in the appropriate roles based on how we use it.
